### ISSUE TITLE
TimeoutException when transcoding m4a files

### ISSUE THREAD

**Issue by issuemover631** [airsonic/airsonic#472]
<a href="https://github.com/arielelkin"><img src="https://avatars3.githubusercontent.com/u/1756909?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [arielelkin](https://github.com/arielelkin)**
_Tuesday Jun 20, 2017 at 18:02 GMT_
_Originally opened as https://github.com/Libresonic/libresonic/issues/472_

----

Libresonic v6.2

Libresonic plays fine any mp3 files, but can't play m4a (ALAC) files on the Web player. The player is updated with the track's duration, but playback won't start. 

Here are the logs: 

Google Chrome 58.0.3029.110
https://pastebin.com/raw/tkdip7aY

Safari 10.1.1
https://pastebin.com/raw/q1dHPYug

**Comment by muff1nman** [comment]
Reopening this @arielelkin as I think its something we need to look into.

**Comment by arielelkin** [comment]
Thanks @muff1nman, I think so too.

If you'd like to test out M4A (ALAC) files, you can find hundreds for free on Bandcamp:
https://bandcamp.com/tag/free-download

You can also find AAC, Ogg Vorbis, WAV and AIFF

But if you want to test on your own stuff, `ffmpeg` is your friend too:
`ffmpeg -i "$filename" -f m4a "${filename%.*}.m4a"`

Let me know if I can help with anything else.

**Comment by arielelkin** [comment]
hi folks! any updates on this?

**Comment by arielelkin** [comment]
Thanks, but this doesn't work for me as I need to also update an iTunes library.

**Comment by jooola** [comment]
#1035 has been merged, might fix this issue.

**Comment by 3Dscrewer** [comment]
No, it is not fixed, playing ALAC Files is still a problem. I'm trying to change from subsonic to airsonic. Subsonic is showing the wrong time in the playlist and it remains when playing but the indicator of the playing position jumps back several times and subsonic is playing until the end.
Airsonic is showing the right play time in the playlist (so far) but once it is starting it is showing the wrong time at the player (Bottom Line). If the player is reaching the wrong time it stops suddenly.
I have tons of ALAC because iTunes, I can't switch to FLAC ...
I can create Test Acounts in both Systems if it helps ...
